docs/pages/management/admin/troubleshooting.mdx

```diff
@@ -18,6 +18,29 @@ services such as the Application Service and Database Service.
 
 ## Step 1/3. Enable verbose logging
 
+<Tabs>
+<TabItem label="With Debug service">
+(!docs/pages/includes/diagnostics/debug-service-intro.mdx!)
```

Comment:
From what I gather, the Debug Service does the same thing as enabling verbose logging. Is that true? If not, these instructions should make it clear what starts happening when the Debug Service runs.

Reply:
In this context, yes. I've updated both places that have a debug service intro so they can be more focused on the use case.